mopidy/install.sh

```diff
@@ -31,10 +34,16 @@ warning() {
 
 # Update apt and install dependencies
 inform "Updating apt and installing dependencies"
-apt update
-apt install -y python3-rpi.gpio python3-spidev python3-pip python3-pil python3-numpy libopenjp2-7
+sudo apt update
+sudo apt install -y python3-spidev python3-pip python3-pil python3-numpy python3-lgpio python3-virtualenvwrapper virtualenvwrapper libopenjp2-7
```

Comment:
I ran `sudo apt-get update && sudo apt-get upgrade -y` before running `install.sh` so maybe this is why I get an error: `ModuleNotFoundError: No module named 'gi'`. 

`sudo apt install -y python3-spidev python3-pip python3-pil python3-numpy python3-lgpio python3-virtualenvwrapper virtualenvwrapper libopenjp2-7 python3-gi`

-----------

However, after doing that, I received an error saying `ValueError: Namespace Gst not available`. Perhaps this is because I installed a raspberry pi lite version instead of the desktop? 

Anyway, adding this fixed that (I'm not sure if we need everything listed here): 
```
sudo apt-get install -y libgstreamer1.0-dev \
     libgstreamer-plugins-base1.0-dev \
     libgstreamer-plugins-bad1.0-dev \
     gstreamer1.0-plugins-ugly \
     gstreamer1.0-tools \
     gstreamer1.0-gl \
     gstreamer1.0-gtk3
```

So combining it all:

`sudo apt install -y python3-spidev python3-pip python3-pil python3-numpy python3-lgpio python3-virtualenvwrapper virtualenvwrapper libopenjp2-7 python3-gi libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-ugly gstreamer1.0-tools gstreamer1.0-gl gstreamer1.0-gtk3`